scripts/components/fancy-divider/fancy-divider.js

```diff
@@ -39,7 +39,7 @@ export const FancyDivider = (props) => {
 				additionalClasses='es-nested-bg-cool-gray-450 es-nested-rounded-1 es-nested-color-pure-white! es-has-enhanced-contrast-icon'
 			/>
 
-			<div className='es-w-full es-flex-1 es-h-px es-bg-cool-gray-100'></div>
+			<div className='es-w-full es-flex-1 es-h-px es-bg-cool-gray-100' />
```

Comment:
All previous changes in this manner have this written as the full tag, why the exception here?

Reply:
in jsx you can write this this way it doesn't really mater

Reply:
I know, but in some places in this PR this approach has reversed logic... just wondering why the inconsistencies 😄

Reply:
It's an linter error now 😄
"Empty components are self-closing"